controllers/sriovoperatorconfig_controller.go

```diff
@@ -229,12 +230,13 @@ func (r *SriovOperatorConfigReconciler) syncWebhookObjs(ctx context.Context, dc
 		data.Data["CaBundle"] = os.Getenv("WEBHOOK_CA_BUNDLE")
 		data.Data["DevMode"] = os.Getenv("DEV_MODE")
 		data.Data["ImagePullSecrets"] = GetImagePullSecrets()
-		external, err := utils.IsExternalControlPlaneCluster(r.Client)
-		if err != nil {
-			logger.Error(err, "Fail to get control plane topology")
-			return err
+
+		data.Data["ExternalControlPlane"] = false
+		if utils.ClusterType != utils.ClusterTypeOpenshift {
```

Comment:
I think this should be `==` ?

@bn222 Were you referring to having this check around the controllers? Or putting it inside `IsHypershift()`?
Something like:

```
func (c *OpenshiftContext) IsHypershift() bool {
        if c == nil {
            return false
        }
	return c.OpenshiftFlavor == OpenshiftFlavorHypershift
}
```

Reply:
Yes, let's not repeat code all over the place. _anything_ related to checking if we are openshift or not, should be in 1 place.